src/ceph.in

```diff
@@ -712,7 +712,7 @@ def main():
         return 1
     except Exception as e:
         print >> sys.stderr, 'Error connecting to cluster: {0}'.\
-            format(e.__class__.__name__)
+             format(e.message)
```

Comment:
Running it with Python 2.6 would get you a deprecation warning:

    DeprecationWarning: BaseException.message has been deprecated as of Python 2.6